pkg/certifier/scorecard/mock/scorecard.go

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by MockGen. DO NOT EDIT.
+// Source: /guac/pkg/certifier/scorecard/types.go
+
+// Package mocks is a generated GoMock package.
```

Comment:
quick question: is file generation covered by `go generate`?

Reply:
This is generated by mockgen.

Reply:
gotcha! Can we add this to [make generate](https://github.com/guacsec/guac/blob/main/Makefile#LL83-L84C19) and that will be checked automatically by the CI for `make generate_up_to_date`! Thanks!

Reply:
This will require https://github.com/golang/mock which needs to be run localy. How do you want to handle the dependency for this tool?

Reply:
Ok, Thanks